docs/lighthouse-ci-configuration.md

```
@@ -223,13 +331,27 @@ lhci collect --config=./.lighthouserc.json
 ## Resources
 
 - [Lighthouse CI Documentation](https://github.com/GoogleChrome/lighthouse-ci)
 - [Vite Preview Server](https://vitejs.dev/guide/cli.html#vite-preview)
 - [Lighthouse Metrics](https://web.dev/lighthouse-performance/)
 - [treosh/lighthouse-ci-action](https://github.com/treosh/lighthouse-ci-action)
+- [Configuration Fix Details](./lighthouse-ci-fix.md)
 
 ---
 
-_Last Updated: October 2025_
+_Last Updated: October 16, 2025_
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)